hbase-server/src/main/java/org/apache/hadoop/hbase/mob/DefaultMobStoreCompactor.java

```diff
@@ -362,11 +508,375 @@ protected boolean performCompaction(FileDetails fd, InternalScanner scanner, Cel
         abortWriter(mobFileWriter);
```

Comment:
log at debug that we're aborting because there are no cells

Reply:
Fixed.